src/software/python_bindings/pass_utilities.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+#include <pybind11/pybind11.h>
+
+#include "software/ai/passing/pass.h"
+
+Pass createPassFromDict(pybind11::dict pass_dict);
+
+pybind11::dict convertPassToDict(const Pass& pass);
```

Comment:
nit: would be nice to have some java docs here even though its pretty obvious what these functions are doing